assets/app/view/game/corporation.rb

```diff
@@ -124,7 +124,7 @@ def render
       def render_title
         title_row_props = {
           style: {
-            grid: '1fr / auto auto',
+            grid: '1fr / auto auto auto',
```

Comment:
just FYI: `repeat(3, auto)` or `grid-auto-flow: column` would do the same.

Reply:
Nice. I used repeat(auto-fit, auto) to only create two columns for corporations and a third for systems.